boards/BMS/Inc/Io/Io_CellVoltages.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <stdint.h>
+#include <stdlib.h>
+#include "App_SharedExitCode.h"
+
+/**
+ * Read cell voltages (100µV) from the cell monitoring chips.
```

Comment:
I'm confused why the `(100uV)` is in the comments here? It doesn't really seem relevant for the user of these functions to know what units the voltages are read in?

Reply:
Alright this should be fixed